addons/mail/static/tests/message/message_tests.js

```diff
@@ -672,6 +673,24 @@ QUnit.test("should not be able to reply to temporary/transient messages", async
     await contains(".o-mail-Message [title='Reply']", { count: 0 });
 });
 
+QUnit.test("squashed transient message should not have date in the sidebar", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Channel 1", channel_type: "chat" });
```

Comment:
`channel_type: "chat"` looks useless, can be omitted and rely on default `channel_type: "channel"`